doc/cli/operator-sdk_test_local.md

```diff
@@ -21,11 +21,12 @@ operator-sdk test local <path to tests directory> [flags]
       --kubeconfig string             Kubeconfig path
       --local-operator-flags string   The flags that the operator needs (while using --up-local). Example: "--flag1 value1 --flag2=value2"
       --molecule-test-flags string    Additional flags to pass to molecule test
-      --namespace string              If non-empty, single namespace to run tests in
       --namespaced-manifest string    Path to manifest for per-test, namespaced resources (e.g. RBAC and Operator manifest)
       --no-setup                      Disable test resource creation
+      --operator-namespace string     Namespace where the operator will be deployed, CRs will be created and tests will be executed (By default it will be in the default namespace defined in the kubeconfig)
       --skip-cleanup-error            If set as true, the cleanup function responsible to remove all artifacts will be skipped if an error is faced.
       --up-local                      Enable running operator locally with go run instead of as an image in the cluster
+      --watch-namespace string        (only valid with --up-local) Namespace where the operator watches for changes. Explicitly set to empty string to watch all namespaces (defaults to the operatorNamespace).
```

Comment:
We have not more up-local. Also, is not it valid for the test as well?

Reply:
@joelanford in a comment asked us to restrict it to `--up-local`.
if `--up-local` is not there then the `WATCH_NAMESPACE` is specified as a environment variable in `deploy/operator.yaml`

ref: https://github.com/operator-framework/operator-sdk/pull/2149#discussion_r349138337

Reply:
Up local no longer exist. We have now operator-sdk run --local.